cva6/README.md

```diff
@@ -51,7 +51,7 @@ The default values are:
 
 - `CVA6_REPO`: `https://github.com/openhwgroup/cva6.git`
 - `CVA6_BRANCH`: `cva6_reorg`
-- `CVA6_HASH`: `95b1070a76d2399ff4a95af70082514b0d0c5743`
+- `CVA6_HASH`: `bb9821d85f88d4e615bfa9bb35c9a78ed6f0b579`
```

Comment:
Hi @ASintzoff, I suggest that we do not place these specific hash values in the README.  This creates a maintenance issue since both the README and the install script need to be kept consistent.  The README should indicate that a specific hash can be selected with an environment variable and that the install script sets the default.

Reply:
Good point. I just force-pushed a fixed version of the README.

Reply:
Thanks!